File: web-service/yacontest-helper/src/main/java/org/ilyatyamin/yacontesthelper/autoupdate/repository/UpdateTaskRepository.java
package org.ilyatyamin.yacontesthelper.autoupdate.repository;

import org.ilyatyamin.yacontesthelper.autoupdate.dao.UpdateTaskDao;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface UpdateTaskRepository extends JpaRepository<UpdateTaskDao, Long> {
}

File: web-service/yacontest-helper/src/main/java/org/ilyatyamin/yacontesthelper/autoupdate/repository/UpdateTaskRepository.kt
package org.ilyatyamin.yacontesthelper.autoupdate.repository

import org.ilyatyamin.yacontesthelper.autoupdate.dao.UpdateTaskDao
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
interface UpdateTaskRepository : JpaRepository<UpdateTaskDao, Long> {
    fun findAllByOwnerId(ownerId: Long): List<UpdateTaskDao>
}
